### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by carl0smat3us** [community/community#68432] [Repositories] [answered]
### Select Topic Area

Bug

### Body

**Issue Report: Persistent Error Accessing Repository Files**

**Problem Description:**
I consistently encounter an error when attempting to access files within repositories. An error message is displayed, indicating that the page is experiencing issues and accompanied by the following error code: `SIGSEGV`.

**Issue Details:**
- **Incident Date:** September 27, 2023
- **Affected Repository:** All
- **Expected Behavior:** I expect to access and open files within the repository seamlessly without needing to refresh the page.
- **Current Behavior:** To access and open files within the repository, I am required to refresh the page.
- **Steps to Reproduce:** The issue can be replicated by simply trying to open directories or files within a repository on the GitHub website.

**Browser:** Microsoft Edge Version 119.0.2109.1

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @carl0smat3us ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub feature Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by Giangpro89** [reply]
Thankyou
